<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>chai lab</title>
 <style> {display:none !important;}
 html * {font-size: 1em !important; font-family: Arial !important;}
 a {text-decoration:none;}</stylei>
<style type="text/css"> a:link{color:blue;} a:visited{color:purple;}a:hover{color:green;}a:active{color:red;}</style>
</head>
<!--
<body>[<a href="https://gene.hanyang.ac.kr/">chai lab (adv.mol.gen.lab)</a>]<br>
-->

<br>top:&nbsp;
<a href="http://www.ncbi.nlm.nih.gov/Sitemap/index.html">ncbi</a>&nbsp;
<a href="http://www.expasy.org/">expasy</a>&nbsp;
<a href="https://flipboard.com/">flipboard</a>&nbsp;
<a href="https://information.hanyang.ac.kr/#/eds/ejournal/result?keyword=nature">ejournal</a>&nbsp;
<a href="http://cafe.naver.com/aprl.cafe">amgl-cafe</a>&nbsp;
<a href="http://www.google.com/ncr">google</a>&nbsp;

<br>hanyang univ.:&nbsp;
<a href="http://www.hanyang.ac.kr/">hanyang</a>&nbsp;
<a href="http://portal.hanyang.ac.kr/">ezhub</a>&nbsp;
<a href="https://learn.hanyang.ac.kr/">learn</a>&nbsp;
<a href="https://cdp.hanyang.ac.kr/">cdp</span></a>&nbsp;
<a href="http://ericaresearch.hanyang.ac.kr/">research</a>&nbsp;
<a href="http://www.sgs.hanyang.ac.kr/">graduate</a>&nbsp;
<a href="https://larc.hanyang.ac.kr:476/">iacuc</a>&nbsp;
<a href="https://check.hanyang.ac.kr/">attendance</a>&nbsp;

<br>chai's lab:&nbsp;
<a href="https://docs.google.com/document/d/1hAZFe5dXtwgXVcvvWHb7XQsmFF6Rh97SHpaKBtgK7Qc/edit?usp=sharing">professor</a>&nbsp;
<a href="https://docs.google.com/spreadsheets/d/105y6rK6SInNxWnmIw--XpV3xBdFi8Yc7du93dwO3p2c/edit?usp=sharing">members</a>&nbsp;
<a href="https://docs.google.com/spreadsheets/d/1avT7loKM3bIoJQnJJf9Vx8wzmGtClQzgDqD4RRX6_b8/edit?usp=sharing">alumni</a>&nbsp;
<a href="https://docs.google.com/document/d/1gl15mBdlcoeS9UCBciYmTgnnVt2xwxSt_JnfUEmwDfw/">papers</a>&nbsp;


<br>journals:&nbsp;
<a href="http://www.isiknowledge.com/">isi</a>&nbsp;
<a href="https://jcr.incites.thomsonreuters.com/">incites</a>&nbsp;
<a href="http://www.cell.com/">cell</a>&nbsp;
<a href="http://www.cell.com/cell-stem-cell/">csc</a>&nbsp;
<a href="http://www.nature.com/">nature</a>&nbsp;
<a href="http://www.scienceonline.org/">science</a>&nbsp;
<a href="http://www.pnas.org/">pnas</a>&nbsp;
<a href="http://www.jbc.org/">jbc</a>&nbsp;

<br>libraries:&nbsp;
<a href="http://information.hanyang.ac.kr/">erica-library</a>&nbsp;
<a href="http://library.hanyang.ac.kr/">seoul-library</a>&nbsp;
<a href="http://www.ncbi.nlm.nih.gov/pubmed/">pubmed</a>&nbsp;
<a href="http://www.kisti.re.kr/">kisti</a>&nbsp;
<a href="http://www.scopus.com/">scopus</a>&nbsp;
<a href="http://www.uspto.gov/">patent</a>&nbsp;
<a href="http://www.european-patent-office.org/">epo</a>&nbsp;
<a href="http://www.kipo.go.kr/">kipo</a>
<a href="http://www.freepatentsonline.com/">uspatents</a>&nbsp;

<br>bioportals:&nbsp;
<a href="http://www.cdc.gov/">cdc</a>&nbsp;
<a href="http://www.nih.gov/">nih</a>&nbsp;
<a href="http://www.fda.gov/">fda</a>&nbsp;
<a href="http://www.nlm.nih.gov/">nlm</a>&nbsp;
<a href="http://www.bioin.or.kr">bioin</a>&nbsp;
<a href="http://www.ibric.org/">bric</a>&nbsp;
<a href="http://www.bio.com/">bio</a>&nbsp;
<a href="http://www.biospace.com/">biospace</a>&nbsp;

<br>resources:&nbsp;
<a href="http://www.ncbi.nlm.nih.gov/">ncbi</a>&nbsp;
<a href="http://www.ncbi.nlm.nih.gov/Sitemap/index.html">ncbi-map</a>&nbsp;
<a href="http://www.genecards.org/">genecards</a>&nbsp;
<a href="http://genome.ucsc.edu/">ucsc</a>&nbsp;
<a href="http://genome-cancer.ucsc.edu/">ucsc-cancer</a>&nbsp;
<a href="http://www.genome.gov/10000375/">nhgri</a>&nbsp;
<a href="http://primer3plus.com/">primer</a>&nbsp;
<a href="http://www.protocol-online.org/">protocol</a>&nbsp;

<br>supporters:&nbsp;
<a href="http://projectreporter.nih.gov/">nih</a>&nbsp;
<a href="http://www.kistep.re.kr/">kistep</a>&nbsp;
<a href="http://www.ipet.re.kr/">ipet</a>&nbsp;
<a href="http://www.cdc.go.kr/">cdc</a>&nbsp;
<a href="http://www.htdream.kr/">htdream</a>&nbsp;
<a href="http://www.keit.re.kr/">keit</a>&nbsp;
<a href="http://www.kiat.or.kr/">kiat</a>&nbsp;
<a href="http://www.nrf.re.kr/">nrf</a>&nbsp;
<a href="http://ernd.nrf.re.kr/">nrf-ernd</a>&nbsp;
<a href="http://www.smba.go.kr/">smba</a>&nbsp;
<a href="http://www.smtech.go.kr/">smtech</a>&nbsp;
<a href="http://www.ernd.go.kr/">keit-ernd</a>&nbsp;

<br>societies:&nbsp;
<a href="http://www.aaas.org/">aaas</a>&nbsp;
<a href="http://www.faseb.org/">faseb</a>&nbsp;
<a href="http://www.iscb.org/">iscb</a>&nbsp;
<a href="http://www.isscr.org/">isscr</a>&nbsp;
<a href="http://web.sfn.org/">sfn</a>&nbsp;

<br>meetings:&nbsp;
<a href="https://meetings.cshl.edu/meetingshome.aspx">cshl</a>&nbsp;
<a href="http://www.keystonesymposia.org/">keystone</a>&nbsp;
<a href="http://www.grc.org/">gordon</a>&nbsp;
<a href="http://www.isscr.org/meetings-events/annual-meetings/isscr-annual-meeting-2019">isscr</a>&nbsp;
<a href="http://www.asbmb.org/meeting2019/">asbmb</a>&nbsp;
<a href="https://www.iscb.org/ismbeccb2019">ismb</a>&nbsp;
<a href="http://www.sfn.org/Meetings/Neuroscience-2019/">sfn</a>&nbsp;
<a href="http://epigenie.com/conferences">epigenome</a>&nbsp;
<a href="http://https://www.abcam.com/index.html?pageconfig=tradeshowsearch&intClassID=5">epigenetics</a>&nbsp;
<a href="https://www.aacr.org/Meetings/Pages/MeetingDetail.aspx?EventItemID=174">aacr</a>&nbsp;
<a href="http://www.bitcongress.com/cancer2019/">bit</a>&nbsp;

<br>news:&nbsp;
<a href="http://www.sciencedaily.com/">sciencedaily</a>&nbsp;
<a href="http://www.eurekalert.org/">eurekalert</a>&nbsp;
<a href="http://www.nature.com/news/">nature</a>&nbsp;
<a href="https://www.genengnews.com/">gen</a>&nbsp;
<a href="https://www.the-scientist.com/">scientist</a>&nbsp;
<a href="http://www.genome.gov/27532724">genometv</a>&nbsp;

<br>companies:&nbsp;
<a href="http://www.addgen.org/">addgene</a>&nbsp;

<br>links:&nbsp;
<a href="http://www.hibrain.net/">hibrain</a>&nbsp;
<a href="http://gmail.com/">gmail</a>&nbsp;
<a href="http://drive.google.com/">drive</a>&nbsp;
<!-- <a href="http://gen.lib.rus.ec/">books</a>&nbsp;-->
<a href="http://en.wikipedia.org/wiki/Main_Page/">wikipedia</a>&nbsp;
<a href="https://gene.hanyang.ac.kr/memo.html">memo</a>&nbsp;<br>


<br>bioinformtics:&nbsp;
<a href="http://scifeeds.com/category/life-science/bioinformatics/">bioinformatics</a>
<a href="http://genome.ucsc.edu/" target="_blank">ucsc</a>&nbsp;
<a href="http://www.genboree.org/epigenomeatlas/index.rhtml">atlas</a>&nbsp;
<a href="http://www.epigenomebrowser.org/">epi-browser</a>&nbsp;
<a href="http://useast.ensembl.org/info/docs/funcgen/regulation_sources.html" target="_blank">ensemble</a>&nbsp;
<a href="http://www.ncbi.nlm.nih.gov/geo/" target="_blank">omnibus</a>&nbsp;
<a href="http://www.genome.gov/26524507">encode</span></a>&nbsp;
<a href="https://www.gencodegenes.org/">genecode</a>&nbsp;
<a href="https://github.com/stephenturner/oneliners">bash</a>&nbsp;

<br>epigenetics:&nbsp;
<a href="http://scifeeds.com/category/life-science/epigenetics/">epigenetics</a>&nbsp;
<a href="http://www.roadmapepigenomics.org/">roadmap</span></a>&nbsp;
<a href="http://www.genboree.org/java-bin/login.jsp">genboree</a>&nbsp;
<a href="http://www.epigenesys.eu/">epigenesys</a>&nbsp;
<a href="http://ihec-epigenomes.net/">ihec</a>&nbsp;
<a href="http://www.epigenie.com/">epigenie</a>&nbsp;
<a href="http://commonfund.nih.gov/epigenomics/" target="_blank">funds</a>&nbsp;

<br>transcriptomics:&nbsp;
<a href="http://www.rna-seqblog.com/" rel="tag">rna-seq</a>&nbsp;
<a href="http://ccb.jhu.edu/software/tophat/index.shtml">tophat</a>&nbsp;
<a href="http://cufflinks.cbcb.umd.edu/">cufflinks</a>&nbsp;
<a href="http://support.illumina.com/sequencing/sequencing_software/igenome.html">igenome</a>&nbsp;
<a href="https://github.com/alexdobin/STAR">star</a>&nbsp;
<a href="https://hemberg-lab.github.io/scRNA.seq.course/">single-cell</a>&nbsp;
<a href="https://www.bioconductor.org/help/course-materials/2015/LearnBioconductorFeb2015/B02.1.1_RNASeqLab.html">deseq2</a>&nbsp;
<a href="https://pods.iplantcollaborative.org/wiki/display/TUT/RNA-seq+Tutorial-+HISAT2%2C+StringTie+and+Ballgown">hisat2</a>&nbsp;

<br>genome-wide data:&nbsp;
<a href="http://ihec-epigenomes.net/">ihec</a>&nbsp;
<a href="https://encodeproject.org/">encode</a>&nbsp;
<a href="http://www.blueprint-epigneome.eu/">blueprint</a>&nbsp;


<br>genome edition:&nbsp;
<a href="https://www.addgene.org/crispr/guide/">cas9</a>&nbsp;
<a href="https://www.broadinstitute.org/visuals/cas13-primer">cas13</a>&nbsp;
<a href="https://commonfund.nih.gov/epigenomics/programhighlights">epigenome</a>&nbsp;
<a href="http://www.atcc.org/">atcc</a>&nbsp;
<a href="http://www.antibodyresource.com/">antibody</a>&nbsp;

<br>ngs analysis tools:&nbsp;
<a href="https://github.com/taoliu/MACS/">macs</a>&nbsp;
<a href="http://homer.salk.edu/homer/ngs/">homer</a>&nbsp;
<a href="https://github.com/crazyhottommy/ChIP-seq-analysis">chip-seq</a>&nbsp;
<a href="https://sites.google.com/site/atacseqpublic/">atac-seq</a>&nbsp;
<a href="http://www.bioconductor.org/">bioconductor</a>&nbsp;<br>

<br>enhancer db:&nbsp;
<a href="https://omictools.com/dbsuper-tool">dbsuper</a>&nbsp;
<a href="http://enhanceratlas.org/">enhanceratlas</a>&nbsp;
<a href="http://http://bind-db.huji.ac.il/bindDB/default_new.php">binddb</>&nbsp;
<a href="http://fantom.gsc.riken.jp/5/">fantom</a>&nbsp;
<a href="http://pedagogix-tagc.univ-mrs.fr/remap/">remap</a>&nbsp;
<a href="http://http://zdzlab.einstein.yu.edu/1/hedd.php">hedd</a>&nbsp;

<br>cancer db:&nbsp;
<a href="http://www.cancerindex.org/geneweb/X070601.htm">cancerindex</a>&nbsp;
<!-- <a href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgS_doOtherUser=submit&hgS_otherUserName=ygchai&hgS_otherUserSessionName=hg19%2Dhepg2%2Dnov052018">ucsc-hep</a>&nbsp;-->
<a href="http://promoter.bx.psu.edu/hi-c/index.html">hi-c</a>&nbsp;
<a href="https://www.proteinatlas.org/">protein</a>&nbsp;
<a href="https://gdac.broadinstitute.org">gdac</a>&nbsp;
<a href="https://portal.gdc.cancer.gov/">cancergenome</a>&nbsp;
<a href="https://dcc.icgc.org">icgc</a>&nbsp;
<a href="https://www.ilca-online.org">ilca-online</a>&nbsp;
<a href="https://www.ilca2019.org">ilca2019</a>&nbsp;
<a href="https://www.karger.com/Journal/Home/255487">karger</a>&nbsp;
<a href="https://medlineplus.gov/livercancer.html">medlineplus</a>&nbsp;

<br>ngs data:&nbsp;
<a href="http://gene.hanyang.ac.kr/amgl-ngs-data/">amgl-data</a>&nbsp;
<a href="http://166.104.249.97/">amgl-results-epic-server</a>&nbsp;
<a href="http://gene.hanyang.ac.kr/analysis/">amgl-results-gene-server</a>&nbsp;
<a href="http://www.ingenuity.com/">ipa</a>&nbsp;
<a href="http://166.104.249.95/">epic-browser</a>&nbsp;<br>


<br>-=-=-&nbsp;
<!--
<br>amgl-server:&nbsp;
<a href="https://gene.hanyang.ac.kr/2018/">2018</a>&nbsp;
<a href="https://gene.hanyang.ac.kr/2019/">2019</a>&nbsp;

<a href="https://gene.hanyang.ac.kr/eteam/">eteam</a>&nbsp;
<a href="https://gene.hanyang.ac.kr/eteam/HepG2-AMGL-NGS/Encode-HepG2/">encode-hepg2</a>&nbsp;
-->
<br>cancer db:&nbsp;
<a href="http://www.cancerindex.org/geneweb/X070601.htm">cancerindex</a>&nbsp;
<a href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgS_doOtherUser=submit&hgS_otherUserName=ygchai&hgS_otherUserSessionName=hg19%2Dhepg2%2Dnov072018">ucsc-hepg2</a>&nbsp;
<a href="http://promoter.bx.psu.edu/hi-c/index.html">hi-c</a>&nbsp;
<a href="https://www.proteinatlas.org/">protein</a>&nbsp;
<a href="https://gdac.broadinstitute.org">gdac</a>&nbsp;
<a href="https://portal.gdc.cancer.gov/">cancergenome</a>&nbsp;
<a href="https://dcc.icgc.org">icgc</a>&nbsp;
<a href="https://www.ilca-online.org">ilca-online</a>&nbsp;
<a href="https://www.karger.com/Journal/Home/255487">karger</a>&nbsp;
<a href="https://medlineplus.gov/livercancer.html">medlineplus</a>&nbsp;
<a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">atalas</a>&nbsp;

<br>bioinfo-papers:&nbsp;
<a href="https://www.nature.com/articles/nprot.2012.016">tophat2</a>&nbsp;
<a href="https://www.nature.com/articles/nprot.2016.095">hisat2</a>&nbsp;
<a href="https://www.nature.com/articles/nprot.2013.099">deseq2</a>&nbsp;
<a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2012-13-3-r16">manorm</a>&nbsp;
<a href="https://www.nature.com/articles/nprot.2012.101">macs2</a>&nbsp;
<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0881-8">rna-seq</a>&nbsp;
<a href="https://academic.oup.com/bib/article/18/2/279/2453282">chip-seq</a>&nbsp;

<br>index:&nbsp;
<a href="https://www.gencodegenes.org/human/stats.html">statistics</a>&nbsp;
<a href="https://support.illumina.com/sequencing/sequencing_software/igenome.html">igenome</a>&nbsp;
<a href="https://www.gencodegenes.org/">gencode</a>&nbsp;
<a href="ftp://ftp.ncbi.nlm.nih.gov/genomes/">ncbi</a>&nbsp;
<a href="ftp://ftp.ensembl.org/pub/current_gtf/homo_sapiens/">ensemble</a>&nbsp;


<br>ngs-tools:&nbsp;
<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc">fastqc</a>&nbsp;
<a href="http://www.usadellab.org/cms/?page=trimmomatic">trimmomatic</a>&nbsp;
<a href="http://www.htslib.org/">samtools</a>&nbsp;


<br>rna-seq:&nbsp;
<a href="https://ccb.jhu.edu/software/tophat/index.shtml">tophat2</a>&nbsp;
<a href="http://cole-trapnell-lab.github.io/cufflinks/">cufflinks</a>&nbsp;
<a href="https://ccb.jhu.edu/software/hisat2/index.shtml">hisat2</a>&nbsp;
<a href="https://github.com/alexdobin/STAR">star</a>&nbsp;
<a href="https://bioconductor.org/packages/release/bioc/html/edgeR.html">edger</a>&nbsp;
<a href="https://bioconductor.org/packages/release/bioc/html/DESeq2.html">deseq2</a>&nbsp;
<a href="http://bioinf.wehi.edu.au/featureCounts/">featurecounts</a>&nbsp;
<a href="http://david.ncifcrf.gov/tools.jsp">david</a>&nbsp;
<a href="http://bejerano.stanford.edu/great/public/html/">great</a>&nbsp;
<a href="http://software.broadinstitute.org/gsea/index.jsp">gsea</a>&nbsp;
<a href="http://libgen.iohttp://166.104.249.86/script/rnaseq.html/">rna-seq</a>&nbsp;



<br>chip-seq:&nbsp;
<a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">bowties</a>&nbsp;
<a href="http://broadinstitute.github.io/picard/">picard</a>&nbsp;
<a href="https://github.com/taoliu/MACS">macs2</a>&nbsp;
<a href="http://homer.ucsd.edu/homer/">homer</a>&nbsp;
<a href="https://github.com/taoliu/MACS/wiki/Advanced:-Call-peaks-using-MACS2-subcommands">adv-macs</a>&nbsp;
<a href="https://guangchuangyu.github.io/ChIPseeker/">chipseeker</a>&nbsp;
<a href="http://bcb.dfci.harvard.edu/~gcyuan/MAnorm/MAnorm.htm">manorm</a>&nbsp;
<a href="https://bitbucket.org/young_computation/rose">rose</a>&nbsp;
<a href="https://deeptools.readthedocs.io/en/latest/index.html">deeptools</a>&nbsp;
<a href="https://easeq.net/">easeq</a>&nbsp;
<a href="http://166.104.249.86/scripts/chip-seq.html/">chip-seq</a>&nbsp;

<br>public-db:&nbsp;
<a href="https://www.ncbi.nlm.nih.gov/gds/">ncbi-geo</a>&nbsp;
<a href="https://sra-download.ncbi.nlm.nih.gov/traces/sra28/SRR">sra</a>&nbsp;
<a href="ftp-trace.ncbi.nlm.nih.gov anonymous/sra/sra-instant/reads/ByStudy/sra/SRP/">srp</a>&nbsp;
<a href="ftp-trace.ncbi.nlm.nih.gov anonymous/sra/sra-instant/reads/ByStudy/sra/SRP/">srp</a>&nbsp;
<a href="http://ihec-epigenomes.net/">ihec</a>&nbsp;
<a href="https://encodeproject.org/">encode</a>&nbsp;
<a href="https://www.encodeproject.org/matrix/?type=Experiment">encode-cancer</a>&nbsp;
<a href="https://cancergenome.nih.gov/">tcga</a>&nbsp;
<a href="http://fantom.gsc.riken.jp/data/">fantom</a>&nbsp;
<a href="https://www.ebi.ac.uk/ena">embl</a>&nbsp;

<br>manuals:&nbsp;
<a href="https://nihlibrary.nih.gov/services/bioinformatics-support/online-bioinformatics-tutorials">nih</a>&nbsp;
<a href="https://github.com/ossu/bioinformatics">ossu</a>&nbsp;
<a href="http://manuals.bioinformatics.ucr.edu/home">ucr</a>&nbsp;
<a href="https://www.reddit.com/r/learnprogramming/wiki/faq#wiki_getting_started">programming</a>&nbsp;
<a href="https://girke.bioinformatics.ucr.eud/GEN242/">gen242</a>&nbsp;

<br>education:&nbsp;
<a href="https://www.coursera.org/specialization/genomic-data-science/">coursera</a>&nbsp;
<a href="https://github.com/griffithlab/rnaseq_tutorial/">griffith</a>&nbsp;
<a href="http://chagall.med.cornell.edu/RNASEQcourse/">cornell</a>&nbsp;
<a href="https://galaxyproject.github.io/training-material/">galaxy</a>&nbsp;


<br>workflow:&nbsp;
<a href="https://bioinformatics-core-shared-training.github.io/RNAseq-R/getting-raw-reads.nb.html">sra</a>&nbsp;
<a href="https://bioinformatics-core-shared-training.github.io/RNAseq-R/">rna-seq-analysis</a>&nbsp;
<a href="https://www.bioconductor.org/help/workflows/RNAseq123/">rna-seq123</a>&nbsp;
<a href="http://bioconductor.org/packages/release/bioc/vignettes/ChIPseeker/inst/doc/ChIPseeker.html">chipseeker</a>&nbsp;
<a href="https://www.bioconductor.org/help/workflows/rnaseqGene/">rna-seq-workflow</a>&nbsp;
<a href="http://biocluster.ucr.edu/~rkaundal/workshops/R_feb2016/ChIPseq/ChIPseq.html">chip-seq-workflow</a>&nbsp;
<a href="https://github.com/crazyhottommy/ChIP-seq-analysis">chipsep-analysis</a>&nbsp;
<a href="https://f1000research.com/articles/5-1542/v1">tcga-workflow</a>&nbsp;
<a href="https://github.com/hbctraining/In-depth-NGS-Data-Analysis-Course">12day</a>&nbsp;


<br>scripts:&nbsp;
<a href="http://166.104.249.86/scripts/rnaseq.html/">rnaseq</a>&nbsp;
<a href="http://166.104.249.86/scripts/chipseq.html/">chipseq</a>&nbsp;
<a href="http://github.com/">github</a>&nbsp;

<br>enhancers:&nbsp;
<a href="http://166.104.249.86/bioinfo/results/hs_liver/fantom/">liver</a>&nbsp;
<a href="http://http://zdzlab.einstein.yu.edu/1/hedd.php">hedd</a>&nbsp;

<br>lncrna:&nbsp;
<a href="https://lncipedia.org/">lncipedia</a>&nbsp;
<a href="http://mitranscriptome.org/">mitranscriptome</a>&nbsp;
<a href="https://www.lncrnablog.com/">lncrna</a>&nbsp;
<a href="http://starbase.sysu.edu.cn/">starbase</a>&nbsp;

<br>lncrna_cancer:&nbsp;
<a href="http://www.oncolnc.org/">oncolnc</a>&nbsp;
<a href="https://www.tanric.org/">tanric</a>&nbsp;
<a href="https://biodbnet-abcc.ncifcrf.gov/">db2db</a>&nbsp;
<a href="https://www.synapse.org/#!Synapse:syn300013/files/">tcga_pancancer</a>&nbsp;


<br>networks:&nbsp;
<a href="https://string-db.org/">string</a>&nbsp;



<br>hepg2:&nbsp;
<a href="http://pedagogix-tagc.univ-mrs.fr/remap/index.php?page=ct">transcription-factor</a>&nbsp;
<a href="http://genome.ucsc.edu/ENCODE/dataMatrix/encodeDataMatrixHuman.html">encode-matrix</a>&nbsp;
<a href="https://docs.google.com/presentation/d/1jwfUUgynLEdW15QGWhQDaCx2HM3NtC6oQgDezyluZ9A/edit#slide=id.g4a7da1b9e6_0_0">memo</a>&nbsp;
<a href="https://docs.google.com/spreadsheets/d/1ZLPNq8Vk46xlMw51gLUv9S-T_PKz4Xv1Wb-UdnMIJC4/edit?usp=sharing">ng2016-tab</a>&nbsp;
<a href="http://epic.hanyang.ac.kr/~amgl/team">team</a>&nbsp;
<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE29611">gse29611</a>&nbsp;

<br>encode_hepgs:&nbsp;
<a href="https://docs.google.com/presentation/d/1wXqVvACenXUChH6zMjtXTKYYqCZW4mgFZ3ts0am3e1c/edit#slide=id.p">review</a>&nbsp;
<a href="">memo</a>&nbsp;
<a href=""></a>&nbsp;


<br>meeting2019:&nbsp;
<a href="https://www.keystonesymposia.org/index.cfm?e=web.Meeting.Program&meetingid=1621">k_inflammation</a>&nbsp;
<a href="https://www.keystonesymposia.org/index.cfm?e=web.Meeting.Program&meetingid=1603">k_long-noncoding-rna</a>&nbsp;
<a href="https://www.keystonesymposia.org/index.cfm?e=web.Meeting.Program&meetingid=1607">k_epigenetics</a>&nbsp;
<a href="https://www.nature.com/natureconferences/cs19/index.html">chicago-epigenetics</a>&nbsp;
<a href="https://www.ilca2019.org">ilca2019</a>&nbsp;
<a href="https://meetings.cshl.edu/meetings.aspx?meet=TUMBIO&year=19">biology_cancer</a>&nbsp;
<a href="https://www.embo-embl-symposia.org/symposia/2019/EES19-10/">embl_non-coding</a>&nbsp;


<br>meeting_planning:&nbsp;
<a href="https://www.nature.com/natureconferences/nextgen19/index.html">next-genomics</a>&nbsp;
<a href="http://www.cell-symposia.com/rnas-2019/">regulatory-rna</a>&nbsp;
<a href="https://www.omicsonline.org/conferences-list/hepatocellular-carcinoma">hepatome</a>&nbsp;
<a href="https://www.grc.org/nucleosides-nucleotides-and-oligonucleotides-conference/2019/">oligomers</a>&nbsp;
<a href="https://www.grc.org/cancer-genetics-and-epigenetics-conference/2019/">cancer-genetics</a>&nbsp;
<a href="https://www.biomarker2018.com/index.php/welcome">valencia</a>&nbsp;
<a href="https://www.iscb.org/ismbeccb2019">ismb</a>&nbsp;
<a href="https://meetings.cshl.edu/meetings.aspx?meet=RNATX&year=19">c_rna</a>&nbsp;
<a href="https://meetings.cshl.edu/meetings.aspx?meet=MOET&year=19">c_transription</a>&nbsp;
<a href="http://www.isscr.org/meetings-events/annual-meetings/isscr-annual-meeting-2019">isscr</a>&nbsp;
<a href="https://www.ilca-online.org/events-calendar/">ilca</a>&nbsp;

<br>backups:&nbsp;
<!-- "http://Volumes/YGCHAI/EPIC_NGS/EpiC_NGS_FINAL_DATA/FASTQGZ-EPIC-FILES/rsync-feb192016.sh">FASTQ</a>&nbsp; -->
<a href="http://166.104.249.86/backup.html/">backups</a>&nbsp;

<br>os:&nbsp;
<a href="http://colab.research.google.com/">jupyter</a>&nbsp;
<a href="https://github.com/amix/vimrc">vimrc</a>&nbsp;
<a href="https://explainshell.com">shell</a>&nbsp;

<br>r: &nbsp;
<a href="https://www.statmethods.net/r-tutorial/index.html">r</a>&nbsp;
<a href="https://www.computerworld.com/article/2497143/business-intelligence/business-intelligence-beginner-s-guide-to-r-introduction.html#tk.ctw-infsb">r-intro</a>&nbsp;
<a href="https://www.tutorialspoint.com/r/">r-tutorial</a>&nbsp;
<a href="https://www.rstudio.com/online-learning/">rstudio</a>&nbsp;

<br>google:&nbsp;
<a href="https://www.google.com/ncr">google</a>&nbsp;
<a href="https://www.google.com/gmail/about/">gmail</a>&nbsp;
<a href="https://www.google.com/intl/en/drive/">drive</a>&nbsp;
<a href="https://photos.google.com/">photos</a>&nbsp;
<a href="https://www.google.com/maps/">maps</a>&nbsp;
<a href="https://www.youtube.com/">youtube</a>&nbsp;
<a href="https://docs.google.com/forms/d/1UwHAu1Q155lJOUqHebd5rYb39TqaRxwz7hFFFvKkB5w/edit">form</a>&nbsp;
<a href="https://docs.google.com/spreadsheets/d/1nRcECLu74IE0mQ0hFTRcVjmd43hEXNCqvkKwMvNv7TY/edit#gid=0">sheet</a>&nbsp;
<a href="https://docs.google.com/document/d/1kFwgZBAEofvmVCW571w2dyQ3Md_EB_zeRYGkQToD1_Y/edit">doc</a>&nbsp;
<a href="https://docs.google.com/presentation/d/1vUrC6_Ze7NhHvX1kXPTyTWuKy0loJnLiLLo97_6RgUI/edit#slide=id.p">ppt</a>&nbsp;
<a href="https://sites.google.com/s/1zKs70cr2U_oEUNnCmZzPclMPXvCVuwb_/p/1rHMNi7Lq4BTRQuhN5beomgweQrHRDRwV/edit">site</a>&nbsp;


<br>mis:&nbsp;
<a href="http://libgen.io/">books</a>&nbsp;
<a href="http://80.82.77.83/">scihub</a>&nbsp;
<a href="https://libgen.pw/">journal</a>&nbsp;
<a href="https://www.clien.net/service/">clien</a>&nbsp;
<a href="https://naver.com/">naver</a>&nbsp;
<a href="https://www.tripadvisor.com">trip</a>&nbsp;
<a href="http://www.radioswissclassic.ch/en">radio</a>&nbsp;


<br>bt:&nbsp;
<a href="https://www.nature.com/news">nature</a>&nbsp;
<a href="http://www.begenomics.com/">begenomics</a>&nbsp;


<br>it:&nbsp;
<a href="https://techcrunch.com/">techcrunch</a>&nbsp;
<a href="https://www.wired.com/">wired</a>&nbsp;
<a href="https://www.theverge.com/">verge</a>&nbsp;
<a href="https://www.pcmag.com/">pc</a>&nbsp;
<a href="https://www.macworld.com/">mac</a>&nbsp;
<a href="https://www.macrumors.com/">apple</a>&nbsp;
<a href="https://www.dell.com/">dell</a>&nbsp;

<a href="https://docs.google.com/presentation/d/1iC9TQ4zzovrYfeCqnydRt0P0HqoBSUE8OLTN88krdYA/edit#slide=id.g4c18363591_0_74">hpse</a>&nbsp;


<br>-=-=-&nbsp;<br>

<br>
<a href="https://gene.hanyang.ac.kr/">home</a>&nbsp;
<!-- style added on Dec.28th, 2018 github.com/flazz/vim-colorschemes -->


<style type="text/css">
    /* loading */
    .lt-sk-three-bounce {
      margin: 2px auto;
      width: 100%;
      text-align: center; }
      .lt-sk-three-bounce .lt-sk-child {
        width: 5px;
        height: 5px;
        background-color: #333;
        border-radius: 100%;
        display: inline-block;
        -webkit-animation: lt-sk-three-bounce 1.4s ease-in-out 0s infinite both;
                animation: lt-sk-three-bounce 1.4s ease-in-out 0s infinite both; }
      .lt-sk-three-bounce .lt-sk-bounce1 {
        -webkit-animation-delay: -0.32s;
                animation-delay: -0.32s; }
      .lt-sk-three-bounce .lt-sk-bounce2 {
        -webkit-animation-delay: -0.16s;
                animation-delay: -0.16s; }

    @-webkit-keyframes lt-sk-three-bounce {
      0%, 80%, 100% {
        -webkit-transform: scale(0);
                transform: scale(0); }
      40% {
        -webkit-transform: scale(1);
                transform: scale(1); } 

    @keyframes lt-sk-three-bounce {
      0%, 80%, 100% {
        -webkit-transform: scale(0);
                transform: scale(0); }
      40% {
        -webkit-transform: scale(1);
                transform: scale(1); } 
  </style></body></html>





